lib/component/statics.js

```diff
@@ -44,42 +27,3 @@ exports.prop = function(name, options){
   this.props[name] = options;
   return this;
 };
-
-/**
- * Connect to channels
```

Comment:
`channel`s are being removed? Is this because components will now be able to access the `scene` object directly?